src/vscode-bicep/src/test/e2e/runTests.ts

```diff
@@ -28,6 +28,8 @@ async function go() {
       extensionTestsPath: path.resolve(__dirname, "index"),
       launchArgs: ["--enable-proposed-api"],
     });
+
+    process.exit(0);
```

Comment:
VSCode kept hanging in macOS after all tests finish running even it showed exit code 0 (see this cancelled pipeline: https://github.com/Azure/bicep/runs/1410368860?check_suite_focus=true). I spent hours tweaking the test environment and trying different vscode versions...but somehow calling `process.exit(0)` explicitly solved it.